### ISSUE TITLE
Recently stopped working

### ISSUE THREAD

**Issue by VelizarVESSELINOV** [alanhamlett/pip-update-requirements#18]
`pip3 list --outdated` provides plenty packages that need to be updated
```
Package           Version Latest Type 
----------------- ------- ------ -----
gunicorn          19.8.1  19.9.0 wheel
pandas            0.23.1  0.23.3 wheel
plotly            2.7.0   3.1.0  sdist

```

`pur -r requirements.txt`
Output: `All requirements up-to-date.`
Which is not correct

`requirements.txt` content example:
```
gunicorn==19.8.1
pandas==0.23.1
plotly==2.7.0
```

**Comment by alanhamlett** [comment]
With that example requirements file I get:

```
$ pur -r requirements.txt
Updated gunicorn: 19.8.1 -> 19.9.0
Updated pandas: 0.23.1 -> 0.23.3
Updated plotly: 2.7.0 -> 3.1.0
All requirements up-to-date.
```

Can you share the output of `pip freeze` along with your OS and `python --version`?

**Comment by VelizarVESSELINOV** [comment]
I installed Python 3.7 a few weeks ago but was running pur 3.6. After few weeks `All requirements up-to-date` I started to have doubts that all the Python community is in vacation mode. After reading about recent fixes of pandas and plotly I understood that pur is not working.
After installing pur 3.7 and restarting bash finally started to work again. Not sure which operation broke pur and which one exactly fixed it.
Now pur is working again.
